build.xml

```diff
@@ -1176,6 +1176,24 @@ osname=macosx;processor=x86;processor=x86-64;processor=ppc
     </subant>
   </target>
 
+  <target name="deploy-local" depends="dist" description="deploy snapshot version to local repository">
```

Comment:
Maybe change the target name to `install-local`, and description: `install snapshot ....`. The maven term for this action is `install`, so I think this change would be less confusing.